classes/ActionScheduler_LogEntry.php

```diff
@@ -4,12 +4,37 @@
  * Class ActionScheduler_LogEntry
  */
 class ActionScheduler_LogEntry {
-	protected $action_id = '';
-	protected $message = '';
+	protected $action_id =  '';
+	protected $message =  '';
+	protected $date;
 
-	public function __construct( $action_id, $message ) {
+	/**
+	 * Constructor
+	 *
+	 * @param mixed  $action_id	Action ID
+	 * @param string $message   Message
+	 * @param string $date
```

Comment:
What format does this expect the date string to be in? Also what timezone? Best to explicitly mention that to avoid bugs from inconsistent use.

MySQL formatted date/time string in GMT/UTC timezone will match Subscriptions. Alternatively, we could match the behaviour of `wc_string_to_datetime()` in WC 3.0+: https://github.com/woocommerce/woocommerce/blob/3.1.2/includes/wc-formatting-functions.php#L579:L597

This is a public API, we need to be clear to 3rd parties how to use it. 🙂